_benchmark/reference/commands/run.md

```diff
@@ -1,25 +1,26 @@
 ---
 layout: default
-title: execute-test
+title: run
 nav_order: 65
 parent: Command reference
 grand_parent: OpenSearch Benchmark Reference
 redirect_from:
+  - /benchmark/commands/run/
   - /benchmark/commands/execute-test/
 ---
 
 <!-- vale off -->
-# execute-test
+# run
 <!-- vale on -->
 
-Whether you're using the included [OpenSearch Benchmark workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads) or a [custom workload]({{site.url}}{{site.baseurl}}/benchmark/creating-custom-workloads/), use the `execute-test` command to gather data about the performance of your OpenSearch cluster according to the selected workload. 
+Whether you're using the included [OpenSearch Benchmark workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads) or a [custom workload]({{site.url}}{{site.baseurl}}/benchmark/creating-custom-workloads/), use the `run` command to gather data about the performance of your OpenSearch cluster according to the selected workload. 
 
 ## Usage
 
-The following example executes a test using the `geonames` workload in test mode: 
+The following example tests a run using the `geonames` workload in test mode: 
```

Comment:
Might be more clear to use: The following example performs a quick test using the `geonames` workload in test mode.